lib/routes/nautilus/topics.js

```diff
@@ -1,44 +1,66 @@
 const cheerio = require('cheerio');
 const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+const path = require('path');
 
 module.exports = async (ctx) => {
-    const url = `http://nautil.us/term/f/${ctx.params.tid}`;
-
+    const url = `https://nautil.us/topics/${ctx.params.tid}/`;
     const res = await got.get(url);
     const $ = cheerio.load(res.data);
-    const list = $('article.search-result').get();
-
+    const list = $('.article-list_item-content').get();
     const out = await Promise.all(
-        list.map(async (item) => {
+        // loading 20 articles by default
+        list.slice(0, ctx.params.limit ? parseInt(ctx.params.limit) : 20).map(async (item) => {
             const $ = cheerio.load(item);
-            const title = $('h3 > a').text();
-            const partial = $('h3 > a').attr('href');
-            const address = `http://nautil.us${partial}`;
+            const title = $('h2').text();
+            // changing 'by $author' to '$author' by hardcoding
+            const author = $('.article-list_item-byline > li:nth-child(1)').text().substring(3);
+            const publishedDate = $('.article-list_item-byline > li:nth-child(2)').text();
+            const address = $('a').attr('href');
             const cache = await ctx.cache.get(address);
             if (cache) {
                 return Promise.resolve(JSON.parse(cache));
             }
             const res = await got.get(address);
             const capture = cheerio.load(res.data);
-            capture('div.reco').remove();
-            const banner = capture('div.banner').html();
-            const textWithTime = capture('div.byline').text();
-            const author = capture('div.byline > span:nth-child(2)').text();
-            const contents = banner + capture('div.page-content').html();
+
+            // disturbing contents
+            capture('.article-collection_box').remove();
+            capture('blockquote').remove();
+            capture('.article-author').remove();
+            capture('.article-collection_box text-center').remove();
+            capture('.article-bottom-newsletter_box').remove();
+            capture('.main-post-comments-toggle-wrap').remove();
+            capture('.supported-one').remove();
+
+            // lazyload images
+            capture('img').each((_, e) => {
+                e = capture(e);
+                e.after(
+                    art(path.join(__dirname, 'templates/image.art'), {
+                        src: e.attr('data-src') ?? e.attr('srcset'),
+                    })
+                );
+                e.remove();
```

Comment:
```suggestion
                e.attr('src', e.attr('data-src') ?? e.attr('srcset'));
```
You can replace the value instead of creating a new element and deleting current one.